asv/repo.py

```diff
@@ -181,11 +181,11 @@ def get_repo(conf):
     if conf.dvcs is not None:
         for cls in util.iter_subclasses(Repo):
             if getattr(cls, 'dvcs') == conf.dvcs:
-                return cls(conf.repo, conf.project)
+                return cls(conf.repo, conf.mirror_dir)
```

Comment:
This seems like an API change -- I think if `conf.mirror_dir` is not provided, it should fall back to the value of `conf.project`, whereas it appears that now it will default to (the literal) "project".